### PR TITLE
updates/testing: mark 30.20191014.1 as a barrier

### PR DESCRIPTION
**Pull request by lucab**
Closes: https://github.com/coreos/fedora-coreos-streams/issues/30

### CODE REVIEW

File: updates/testing.json [hunk 1]
```diff
@@ -12,21 +12,11 @@
...
-    },
     {
       "version": "30.20191014.1",
       "metadata": {
-        "rollout": {
-          "start_epoch": 1574881200,
-          "start_percentage": 0.0,
-          "duration_minutes": 1200
+        "barrier": {
+          "reason": "https://github.com/coreos/fedora-coreos-streams/issues/30"
```
Comment (jlebon) [inline updates/testing.json]: Cool, thanks for creating a better targeted issue to use as the reason here!

File: updates/testing.json [hunk 2]
```diff
@@ -12,21 +12,11 @@
         }
       }
     },
-    {
-      "version": "30.20191014.0",
-      "metadata": {
-        "rollout": {
-          "start_percentage": 100
```
Comment (jlebon) [inline updates/testing.json]: Dropping the rollout section entirely is on purpose, right? And to confirm, nodes that check in now would still get it even without it?

Comment (lucab) [reply updates/testing.json]: 1. Yes it's on purpose. Both rollouts are at 100% already, so the oldest one can be safely dropped. The newest one stays, it is just converted to a barrier.
 2. Yes, barriers are active as long as they exist. They don't need an explicit 100% rollout in place to be active.

**Comment by jlebon** [comment]
Will let you merge and sync!

**Comment by lucab** [comment]
Sync'ed.

**Pull request merged**